packages/mdl-fab/README.md

```diff
@@ -0,0 +1,90 @@
+# MDL FAB
+
+The MDL FAB component is a spec-aligned button component adhering to the
+ [Material Design FAB requirements](https://material.google.com/components/buttons-floating-action-button.html).
+ It works without JavaScript with basic functionality for all states.
+ If you initiate the JavaScript object for a button, then it will be enhanced with ripple effects. (Not yet implemented)
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+## Usage
+
+The demonstrations use the [Material Design Icon Font](https://design.google.com/icons/).
+You may include this to use them as shown or use any other icon method you wish.
+
+### Default
+
+```html
+<button class="mdl-fab material-icons" aria-label="Favorite">
+  favorite
+</button>
+```
+
+### Mini
+
+```html
+<button class="mdl-fab mdl-fab--mini material-icons" aria-label="Favorite">
+  favorite
+</button>
+```
+
+### Plain
+
+```html
+<button class="mdl-fab mdl-fab--plain material-icons" aria-label="favorite">
+  favorite
+</button>
+```
+
+### Absolutely positioned
+
+By default the FAB rests in the page where it sits in the content flow.
+Developers must position it as-needed within their applications designs.
+
+```html
+<!--
+  This will position the FAB in the bottom-right corner.
+  Modify to fit your designs requirements.
+-->
+<style>
+.app-fab--absolute.app-fab--absolute {
```

Comment:
Since the generated scripts will be injected after this, the double-class hack is to increase specificity to make sure these rules take place without `!important`. Otherwise the position would get overridden and it would never get moved.